crates/astria-sequencer/src/authority/mod.rs

```diff
@@ -81,3 +82,34 @@ impl ValidatorSet {
             .wrap_err("failed to map one or more astria validators to cometbft validators")
     }
 }
+
+#[derive(Debug, PartialEq, Eq)]
+#[cfg_attr(test, derive(Clone))]
+pub(crate) struct ValidatorNames(BTreeMap<[u8; ADDRESS_LEN], String>);
```

Comment:
would prefer to store each validator name in a separate storage entry so we don't have to fetch every single validator name just to update one name. then to get all the names, you can use a prefix iterator to return a stream, as all validator names will be stored after one prefix (eg. `valname/<encoded_public_key>`.

see example here: https://github.com/astriaorg/astria/blob/3cabf59c1c0de0c6d0985b7363e34e0deddfd568/crates/astria-sequencer/src/fees/state_ext.rs#L371

Reply:
Thanks for this catch! Done in a7a64af